### ISSUE TITLE
jsontogeojson: Add docs

### ISSUE THREAD

**Issue by duncandewhurst** [Open-Telecoms-Data/ofdskit#3]
Please can we add helpstrings for the command and its arguments.

**Comment by odscjames** [comment]
Just to be clear, what do you mean by helpstrings - stuff that comes up when you pass --help to the cli?

Tho yes, any docs here are better than the current first commit!

**Comment by duncandewhurst** [comment]
Yep, I was thinking that if we add the docs directly in the code that will be easier for users and maintainers as the docs will be proximate to the use of the tool and the actual code.

As much as possible, we can then reuse that documentation in the https://ofdskit.readthedocs.io/en/latest/
